src/views/dashboard/index.js

```diff
@@ -25,6 +27,17 @@ class Dashboard extends Component {
   }
 
   render() {
+    const renderUserCreatedMessage = () => (
+      <section className={styles.userCreated}>
+        <strong>YOUR ACCOUNT WAS CREATED</strong>
+        <small>Check your mail for credentials and more information</small>
+      </section>
+    )
+
+    const renderCrawlerStatus = (isLoading) => (
+      <section className={`${styles.crawler} ${isLoading ? styles.crawlerRunning : styles.crawlerDone}`} />
```

Comment:
You should probably make this a regular component that receives`isLoading` prop, inside it you can display that message that's on the css.